app/helpers/application_helper/toolbar_builder.rb

```diff
@@ -599,8 +599,8 @@ def hide_button?(id)
       type = get_vmdb_config.fetch_path(:server, :remote_console_type)
       return type != 'MKS' || !@record.console_supported?(type)
     when "vm_vnc_console"
-      type = get_vmdb_config.fetch_path(:server, :remote_console_type)
-      return type != 'VNC' || !@record.console_supported?(type)
+      # :remote_console_type is only useful for vmware consoles
+      return !@record.console_supported?('vnc')
```

Comment:
100% agree @himdel, my change needs to be limited to VMware or reverted.  If reverting, we need to restore the non-generalized names for the buttons to avoid user confusion (ie: two 'VM Console' buttons).

@skateman, sounds like a great idea for a future PR.  Putting that on my list of to-dos.